src/Eloquent.php

```diff
@@ -85,25 +85,29 @@ public static function writeEloquentModelHelper(Command $command, Filesystem $fi
         }
 
         $filename = $reflection->getFileName();
-        if ($filename) {
-            $contents = $files->get($filename);
-            if ($contents) {
-                $count    = 0;
-                $contents = str_replace($originalDoc, $docComment, $contents, $count);
-                if ($count > 0) {
-                    if ($files->put($filename, $contents)) {
-                        $command->info('Wrote expected docblock to ' . $filename);
-                    } else {
-                        $command->error('File write failed to ' . $filename);
-                    }
-                } else {
-                    $command->error('Content did not change ' . $contents);
-                }
-            } else {
-                $command->error('No file contents found ' . $filename);
-            }
-        } else {
+        if (!$filename) {
             $command->error('Filename not found ' . $class);
+            return;
+        }
+
+        $contents = $files->get($filename);
+        if (!$contents) {
+            $command->error('No file contents found ' . $filename);
+            return;
+        }
+
+        $count = 0;
+        $contents = str_replace($originalDoc, $docComment, $contents, $count);
+        if ($count <= 0) {
```

Comment:
How can `$count` be < 0 when returned from `str_replace`?

Reply:
I thing your misreading between `$count` and `$contents`. Count is an optional parameter, passed by reference, that if passed will hold the number of matched and replaced needles.

Reply:
> Count is an optional parameter, passed by reference, that if passed will hold the number of matched and replaced needles.

Yes, I think I got this right. But it can't be `< 0`, can it? It can be 0 and > 0.

In a sense, you could just write `if (!$count) {` too but I guess `if ($count === 0) {` is the most crystal clear.